src/MyTested.AspNetCore.Mvc.Caching/Builders/Data/MemoryCacheBaseBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Data
+{
+    using Microsoft.Extensions.Caching.Memory;
+    using Microsoft.Extensions.DependencyInjection;
+    using System;
+
+    /// <summary>
+    /// Used for building mocked <see cref="IMemoryCache"/>.
+    /// </summary>
+    public abstract class MemoryCacheBaseBuilder
```

Comment:
It should be `BaseMemoryCacheBuilder` to follow the other base classes convention.